docs/usage.md

```diff
@@ -314,6 +314,8 @@ If `-profile` is not specified, the pipeline will run locally and expect all sof
   - A generic configuration profile to be used with [Charliecloud](https://hpc.github.io/charliecloud/)
 - `apptainer`
   - A generic configuration profile to be used with [Apptainer](https://apptainer.org/)
+- `wave`
+  - A generic configuration profile to enable [Wave](https://seqera.io/wave/) containers. Use together with one of the above (requires Nextflow ` 24.03.0-edge` or later).
```

Comment:
Maxime we need to update this in the template if we rename the profile as in https://github.com/nf-core/rnaseq/pull/1291

Reply:
Indeed, I was thinking the exact same

Reply:
Are we 100% sure with the name?